tests/components/esphome/test_select.py

```diff
@@ -1,9 +1,24 @@
 """Test ESPHome selects."""
 
+from collections.abc import Awaitable, Callable
 from unittest.mock import call
 
-from aioesphomeapi import APIClient, SelectInfo, SelectState
+from aioesphomeapi import (
+    APIClient,
+    EntityInfo,
+    EntityState,
+    SelectInfo,
+    SelectState,
+    UserService,
+    VoiceAssistantFeature,
+)
+
+from homeassistant.components.assist_satellite import (
+    AssistSatelliteConfiguration,
+    AssistSatelliteWakeWord,
+)
 
+# pylint: disable-next=hass-component-root-import
```

Comment:
why do we now need this pylint exception? the code in the next line didnt change right

Reply:
copied from the assist tests. Maybe not needed there either. Will check and remove if thats the case

Reply:
Looks like its needed there but makes not sense here after ruff rearranged the imports. Removed

Reply:
[esphome_quality_round_2 6fbdf27475c] remove # pylint: disable-next=hass-component-root-import
 1 file changed, 2 deletions(-)